### PR TITLE
Hometask2

### PR DESCRIPTION
**Pull request by thetreew**
@ketanthakkar For next tasks, please create pull requests. It will be easier to review, thanks :)

### CODE REVIEW

**Comment by ketanthakkar** [comment]
@thetreew Created pull request. https://github.com/ketanthakkar/ReactMentoring/pull/3

File: src/index.html [hunk 1]
```diff
@@ -1,41 +1,10 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<title>Hello React</title>
-	<script src="https://unpkg.com/react@16/umd/react.development.js"></script>
-    <script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
-    <script src="https://unpkg.com/babel-standalone@6.15.0/babel.min.js"></script>
+    <meta http-equiv="Content-type" content="text/html; charset=utf-8"/>
+	<title><%= htmlWebpackPlugin.options.title %></title>
```
Comment (thetreew) [inline src/index.html]: Indentation issue.

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const UglifyJsPlugin = require("uglifyjs-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const OptimizeCSSAssetsPlugin = require("optimize-css-assets-webpack-plugin");
+
+module.exports = function (env, options) {
+  const isProduction = options.mode === "production";
```
Comment (thetreew) [inline webpack.config.js]: In scripts (in `package.json`) `env` is used and here `mode`.
`"build:dev": "webpack --env dev --progress --profile --colors"`

Please update webpack config to make use of `env`.

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+          sourceMap: true
+        }),
+        new OptimizeCSSAssetsPlugin({})
+      ]
+    } : {},
+
+    module: {
+      rules: [
+      {
+        test: /\.js$/,
```
Comment (thetreew) [inline webpack.config.js]: Are `jsx` files included here as well?

**Comment by thetreew** [comment]
@ketanthakkar There are some vulnerable packages in your dependencies. Please run `npm audit` for details.

**Comment by ketanthakkar** [comment]
@thetreew Done changes. Thank you. :)

File: package.json [hunk 4]
```diff
@@ -24,13 +24,13 @@
...
+    "react-dom": "^16.6.3"
+  },
+  "devDependencies": {
     "react-redux": "^5.1.1",
     "react-router-dom": "^4.3.1",
     "redux": "^4.0.1",
-    "redux-devtools": "^3.4.2"
-  },
-  "devDependencies": {
+    "redux-devtools": "^3.4.2",
```
Comment (thetreew) [inline package.json]: I meant to move `redux-devtools` only. `react-redux`, `react-router-dom` and `redux` should stay in `dependencies`

Comment (ketanthakkar) [reply package.json]: Done. :)

**Pull request merged**